app/src/main/java/com/android/sample/model/recipe/CreateRecipeViewModel.kt

```diff
@@ -73,7 +73,10 @@ class CreateRecipeViewModel(private val repository: FirestoreRecipesRepository)
       val recipe = recipeBuilder.build()
       repository.addRecipe(
           recipe,
-          onSuccess = { _publishError.value = RECIPE_PUBLISHED_SUCCESS_MESSAGE },
+          onSuccess = {
+            _publishError.value = RECIPE_PUBLISHED_SUCCESS_MESSAGE
```

Comment:
Could you please clarify the role of `publishError`? From the name, it sounds like it’s meant for handling errors, but I noticed it also changes on `onSuccess`. This makes its purpose a bit unclear. Is it intended to handle both success and failure states? A bit of context here would really help in understanding how it’s used and might also help in renaming it for clarity if needed. Thanks!

Reply:
Hey! Thanks for the thoughtful feedback! 😊 You’re absolutely right—the publishError is currently handling both success and failure messages, so the name might be misleading. I’ll rename it to publishStatus to better reflect that it’s used for both success and error states.

Thanks again for catching that—it’ll definitely make the code clearer!